### DISCUSSION TITLE
deploy binary in the current folder

### DISCUSSION THREAD

**Discussion by hernanbosqued** [holgerbrandl/kscript#422] [Q&A] [answered]
when i compile my script with the -p parameter the binary is deployed in some chache folder
i.e.: 
```
[kscript] Packaged script 'run_test' available at path:
[kscript] /root/.cache/kscript/package_a0a2a18dee8849fb7b7672255b453ea0/build/libs/run_test
```

there is a way to deploy that binary in the current folder when i m running the command?

thanks

**Comment by aartiPl** [comment, marked as answer]
Hi! No, there is no such option currently. Enhancing kscript with this functionality should not be challenging, so patches are welcome.
